packages/lib-react-components/.storybook/lib/example.md

```diff
@@ -6,7 +6,8 @@
 |anchor|`[link](https://www.zooniverse.org/)`|[link](https://www.zooniverse.org/)|
 |relative link|`[relative-link](/relative)`|[relative-link](/relative)|
 |image|`![imagealttext](https://panoptes-uploads.zooniverse.org/production/subject_location/66094a64-8823-4314-8ef4-1ee228e49470.jpeg)`|![imagealttext](https://panoptes-uploads.zooniverse.org/production/subject_location/66094a64-8823-4314-8ef4-1ee228e49470.jpeg)|
-|image resized|`![imagealttext =100x100](https://panoptes-uploads.zooniverse.org/production/subject_location/66094a64-8823-4314-8ef4-1ee228e49470.jpeg)`|![imagealttext =100x100](https://panoptes-uploads.zooniverse.org/production/subject_location/66094a64-8823-4314-8ef4-1ee228e49470.jpeg)|
+|image resized in alt text|`![imagealttext =100x100](https://panoptes-uploads.zooniverse.org/production/subject_location/66094a64-8823-4314-8ef4-1ee228e49470.jpeg)`|![imagealttext =100x100](https://panoptes-uploads.zooniverse.org/production/subject_location/66094a64-8823-4314-8ef4-1ee228e49470.jpeg)|
```

Comment:
I assume this will be OK, but what happens if my alt text contains an `=` character? eg. 
```html
<img alt='A blackboard showing the expression 2x2=4' … />
```

Maybe use realistic alt text strings in the test cases, even if it's just lorem ipsum?

Reply:
I'll add this example to the storybook example.md.